### PR TITLE
Adds command option to specify generated .ipa filename

### PR DESCRIPTION
**Pull request by egold**
This addresses issue #182, and improves interoperability with other projects that might have expectations about what the .ipa is called.

### CODE REVIEW

File: lib/shenzhen/commands/build.rb [hunk 1]
```diff
@@ -32,6 +33,7 @@
     @xcconfig = options.xcconfig
     @xcargs = options.xcargs
     @destination = options.destination || Dir.pwd
+    @ipa_name_override = options.ipa || nil
```
Comment (KrauseFx) [inline lib/shenzhen/commands/build.rb]: As far as I know, `options.ipa` will be nil by default if it's not given by the user

**Comment by KrauseFx** [comment]
Thanks, this pull request looks great, would love this to be merged :+1: 

It could probably append `.ipa` if no file extension is given.

File: lib/shenzhen/commands/build.rb (hunk 1 above)
Comment (egold) [inline lib/shenzhen/commands/build.rb]: Good point, fixed that

**Comment by mattt** [comment]
Awesome work, @egold. I really appreciate you taking the time to address this. I'm cutting a new release now. Lots of great things in the next version of Shenzhen!

**Pull request merged**